src/components/icon/js/iconService.js

```diff
@@ -1,5 +1,13 @@
   angular
     .module('material.components.icon' )
+    .constant('svgRegistry', {
```

Comment:
@programmist What do you think about privatizing that a bit more. Something like `$$mdSvgRegistry`?

For example, `$$interimElement` is also an internal service for ngMaterial, so I think the `$$` prefix should be used consistent. Thoughts?